pyicloud/services/photos.py

```diff
@@ -126,6 +126,8 @@ class PhotosService(object):
         },
     }
 
+    ALBUM_TYPES = {"album": 0, "folder": 3}
```

Comment:
If you know other types, you can fill it even if we don't use it now.

Reply:
Unfortunately I don't know what the other types are.